Code.gs

```diff
@@ -18,7 +18,7 @@
 
 var sourceCalendars = [                // The ics/ical urls that you want to get events from along with their target calendars (list a new row for each mapping of ICS url to Google Calendar)
                                        // For instance: ["https://www.calendarlabs.com/ical-calendar/ics/76/US_Holidays.ics", "US Holidays"]
-  ["icsUrl1", "targetCalendar1"],
+  ["icsUrl1", "targetCalendar1"],      //if you want to sync your MAIN/default account number - use your Gmail email address 
```

Comment:
I understand why you might've put this comment here. Let's rephrase a little bit and put on its own line above (right under `// For instance...`

Let's make it say: `// If you want to sync to your main Google Calendar, use your Gmail email address for targetCalendar1`